html_table_parser/parser.py

```diff
@@ -52,7 +56,7 @@ def handle_endtag(self, tag):
             self._in_th = False
 
         if tag in ['td', 'th']:
-            final_cell = " ".join(self._current_cell).strip()
+            final_cell = "".join(self._current_cell).strip()
```

Comment:
My argument for this is that HTML encoding is more common and I did not find a way to handle it both properly. Consider this example:

    <table>
      <tr>
        <td>
          Hello W&#246;rld
        </td>
      </tr>
    </table>